.circleci/ds000210_outputs.txt

```diff
@@ -22,8 +22,6 @@ fmriprep/sub-02/figures/sub-02_T1w_seg_brainmask.svg
 fmriprep/sub-02/figures/sub-02_T1w_t1_2_mni.svg
 fmriprep/sub-02/figures/sub-02_task-cuedSGT_run-01_echo-1_bold_flirt.svg
 fmriprep/sub-02/figures/sub-02_task-cuedSGT_run-01_echo-1_bold_rois.svg
-fmriprep/sub-02/figures/sub-02_task-cuedSGT_run-01_echo-2_bold_rois.svg
```

Comment:
I'm aware of this. CompCor does not make much sense for ME datasets as we plan to integrate MEICA

Reply:
For ME datasets, we're only maintaining the first echo's CompCor for now? Why the first over the second echo, or whichever echo is closest to 30ms?

Reply:
Yes, you are right. I started this before we got your PR merged. I'll also fix this before merging.

The other open thread about ME is fieldmap correction. Making that easy was part of the reasoning under #1017.

Reply:
Okay @emdupre, I've revised this code and it should:
- Enable fieldmap correction on the T2* map
- Generate CompCor masks for the shortest echo (with your PR)
- Generate one _preproc derivative per echo.

Please give it a quick check :)

Reply:
https://github.com/oesteban/fmriprep/blob/39113d9614e6ee62425c074811fd06351cee46a8/fmriprep/workflows/bold/base.py#L526 
It seems like this node is no longer connected ? Unless I'm overlooking it. Is that intended to connect to the confounds wf?

Reply:
Okay, just could check on this. So you extracted the echo for the reports. But now they happen within their generating workflow. So it wasn't necessary anymore. I'll remove this node.